tex/lecture9.tex

```diff
@@ -0,0 +1,295 @@
+bijective correspondence. The result proven above, is sometimes written as:
+
+\[ \Efrac{\lscat C (\phold, A) \natarrow \lscat C (\phold, B) }{ \lscat C(A, B) }.\]
+
+Suppose one was trying to show that two objects $A$ and $B$ are isomorphic.
+This will usually involve constructing some isomorphism $ f : A \to B $.  In
+practice, one can sometimes use the above result and instead come up with some
+natural isomorphism $\lscat C (\phold, A) \natarr{\varphi} \lscat C (\phold,
+B)$ (which amounts to showing $ \Efrac{X \to A}{X \to B} $ `is natural in $X$',
+that is, holds for all $X \in \lscat C^\op $) and then define $f$ to be
+$\widehat{\varphi}$.
+
+\section{A product construction}
+
+Fix a shape: $ A \overset{\alpha}{\longleftarrow} P \morpharr{\beta} B $. 
+Define the functor $ K = \lscat C (\phold, A) \times \lscat C
+(\phold, B)$ (the product of two functors is a a functor) and the infranatural
+transformation $\varphi = \lscat C (\phold, P) \natarrow K $. Hence, for each $X
+\in \lscat C^\op $, we have: 
+
+\begin{center}
+\begin{tikzcd}
+    \lscat C(X, P) \arrow[rr] & & \lscat C (X, A) \times \lscat C (X, B) \\
+    X \arrow[dd, "p", swap] & & X\arrow[dd, "\alpha \circ p", end anchor={[xshift=1em]north west}, swap] \arrow[dd, "\beta \circ p", end anchor ={[xshift=-1em]north east}] \\
+    \arrow[rr, mapsto, "\varphi(p)", start anchor={[xshift=2em]}, end anchor={[xshift=-4em]}] & & \  \\
+    P & & A \hskip 4em B
+\end{tikzcd}
+\end{center}
+
+We now claim that $\varphi = (\alpha \circ \phold, \beta \circ \phold) = \setof{
+{(\alpha \circ \phold, \beta \circ \phold)}_X }_{X \in \lscat C^\op}$ satisifies
+the naturality condition.
+
+% Dhruv: The diagram presented in lectures did not include the C(X,P)/C(Y,P) row
+% or the C(-,P) arrows. It also said (alpha o g, beta o g), which I believe was
+% incorrect, so wrote it as followed.
+
+\begin{center}
+\begin{tikzcd}
+    X \arrow[rr, "{\lscat C (\phold, P)}"]
+        && \lscat C(X, P) \arrow[rr, "{(\alpha \circ\;\phold,\;\beta \circ\;\phold)}_X"]
+            \arrow[dd, "{C(g,P)}"]
+        && K (X) \arrow[dd, "K(g)"]
+        & (a,b) \arrow[dd, mapsto] \\
+    & & & & & \\
+    Y \arrow[uu, "g"] \arrow[rr, "{\lscat C (\phold, P)}"]
+        && \lscat C (Y, P) \arrow[rr, swap, "{(\alpha \circ\;\phold,\;\beta \circ\;\phold)}_Y"]
+        && K(Y)
+        & (a \circ g, b \circ g)
+\end{tikzcd}
+\end{center}
+
+We can determine what $\varphi$ looks like by calculating what $id_P$ is mapped
+to:
+
+\begin{center}
+\begin{tikzcd}
+    \varphi_P : & \lscat C (P, P) \arrow[rr, Rightarrow] && \lscat C (P, A) \times \lscat C (P, B)
+\end{tikzcd}
+\begin{tikzcd}
+              & id_P \arrow[rr, mapsto] && (P \morpharr{\alpha} A,\quad P \morpharr{\beta} B)
+\end{tikzcd}
+\end{center}
+
+\begin{exercise}
+
+% Dhruv: The original phrasing was something along the lines of: "Can we
+% reconstruct phi from A <-(a)- P -(b)-> B but I was unsure what that meant.
```

Comment:
So phi = {phi_X}_X is the family of natural transformations C(-,P) => C(-,A) x C(-,B) that we want to characterise exactly, and I believe the idea is that we want to know whether we can reconstruct _all_ of them from just one?